SoftRenderingApp3D.App/DataStructures/ObjectSerializer.cs

```diff
@@ -0,0 +1,25 @@
+﻿using SoftRenderingApp3D.App.DataStructures;
+using System.Text.Json;
+using System.IO;
+
+
+namespace SoftRenderingApp3D.App.DataStructures  {
+    public class ObjectSerializer
+    {
+        public void SerializeObjectToFile(DisplayModelData model, string filePath)
```

Comment:
Serializer should probably have a generic object as an input:
SerializeObjectToFile(object model, string filePath)